decidim/decidim
# Remove `foundation_rails_jelper` gem

#### :tophat: What? Why?
Since we want to start the rails 7.1 upgrade and the `foundation_rails_helper` gem is not being upgraded, we integrate it with the application. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #?
- Fixes #?

#### Testing
1. Switch to the branch, start your dev server 
2. Browse and observe the forms
3. Compare it with the nightly 
4. See there are no changes

:hearts: Thank you!
